### ISSUE TITLE
Issues with NMODL as submodule and use of sympy

### ISSUE THREAD

**Issue by iomaganaris** [BlueBrain/CoreNeuron#278]
When `CoreNeuron` is installed with `NMODL` as submodule it's not setting the `PYTHONPATH` that includes the `NMODL` python module path and its dependencies (for example _sympy_) to be able to run `NMODL`.
Branch `fix/nmodl_submodule` attempts to fix this issue. Should also be fixed if `CoreNeuron` and `NMODL` are installed as submodule of `Neuron`.
Then there is another issue when building `CoreNeuron` with `NMODL` as submodule in parallel. In this case the execution of `nrnivmodl-core` in `CoreNeuron` starts before the generation of the `NMODL` python module, so the execution of `nrnivmodl-core` fails.
Tried to tackle that on the same branch but didn't manage to fix it.
